lib/std/math/big/int.zig

```diff
@@ -127,7 +127,7 @@ pub const Int = struct {
         self.limbs = try self.allocator.?.realloc(self.limbs, capacity);
     }
 
-    fn assertWritable(self: Int) void {
+    pub fn assertWritable(self: Int) void {
```

Comment:
With https://github.com/ziglang/zig/pull/4968, `assertWritable` is now not used outside of `lib/std/math/big/int.zig`. Would it make sense to make this private again?

Reply:
Yeah. Further, there are a _lot_ of places this PR doesn't catch and updating them all is going to be risky since I'm not familiar with the std sources personally. I think @andrewrk would have to step in and manhandle some stuff after merging.